policybot/pkg/config/args.go

```diff
@@ -66,6 +66,27 @@ type Nag struct {
 	Message string
 }
 
+type FlakeChaser struct {
+	// Name of the nag.
+	Name string
+
+	// InactiveDays represents the days that a flakey test issue hasn't been updated.
+	InactiveDays int
+
+	// CreatedDays determines the bot search range, only issues created within this days ago
+	// are considered.
+	CreatedDays int
+
+	// Message is the message the bot comments on flaky test issues.
+	Message string
+
+	// DryRun determines whether we post updates to the issues.
+	DryRun bool
+
+	// Repo determines the repo this bot is applied on.
+	Repo string
```

Comment:
And nesting this as mentioned above, you don't need this string.

Note that repo names are org-specific, so it's not enough to have a naked repo name. It always needs to be done in the context of an org.

Reply:
Done. Now the repos is a map with fully quilified github uri. Add config yaml as well.